rails_application/app/controllers/orders_controller.rb

```diff
@@ -74,12 +74,13 @@ def remove_item
   end
 
   def create
-    ApplicationRecord.transaction { submit_order(params[:order_id], params[:customer_id]) }
-    redirect_to order_path(params[:order_id]), notice: "Your order is being submitted"
-  rescue Ordering::Order::IsEmpty
-    redirect_to edit_order_path(params[:order_id]), alert: "You can't submit an empty order"
-  rescue Crm::Customer::NotExists
-    redirect_to order_path(params[:order_id]), alert: "Order can not be submitted! Customer does not exist."
+    Orders::SubmitService.new(order_id: params[:order_id], customer_id: params[:customer_id]).call.then do |result|
```

Comment:
I initially wanted to have two separate for the admin and client side but then they look almost entirely the same. So I decided to merge them so we don't duplicate too much code and tests.